### ISSUE TITLE
caps2esc doesn't work with windows key

### ISSUE THREAD

**Issue by leos** [oblitum/Interception#51]
If I run the caps2esc sample and try to do the desktop switch shortcut of `Win+Ctrl+Right Arrow` by using the CapsLock key for Ctrl what I actually get is `Win+Right Arrow` which moves my window over to the right half of the screen. Doing the shortcut with the native Ctrl key works properly.

Is this a known issue? ie. combinations with the Win key don't work properly or something along those lines?

**Comment by oblitum** [comment]
It's a known issue with the caps2esc sample (not the library/driver). When on Windows I use that shortcut a lot to swap multiple desktops but I didn't bother taking a look at how to fix it since, actually, the actul ctrl for that shortcut is easier for me than using the capslock key.

**Comment by oblitum** [comment]
As you have bring the subject up, I'm gonna take a look at this soon.

**Comment by leos** [comment]
Thanks for looking at it Francisco! I agree its more convenient for this shortcut, but I'm trying to build muscle memory to just use the caps lock as ctrl.

Let me know if there's anything I can do to help debug.

**Comment by oblitum** [comment]
This issue doesn't happen with the Linux caps2esc version, which is implemented using the same logic. I also recalled that I've reasoned about this some time ago, the thing is that these new Windows shortcuts seem to be implemented in some special way internal to Windows that render the library/driver ineffective to them, so actually I was mistaken, Interception can't circumvent these shortcuts sadly. I'll confirm further and in any case I discover anything I'll reopen the issue but I have no ideas for now, so closing it as wontfix.
